pep-9999.rst

```diff
@@ -138,6 +138,19 @@ the stub definition should be::
   from typing import ContextManager
   def f() -> ContextManager[int]: ...
 
+Positional-only arguments
```

Comment:
Because of changes to how typed-ast is developed, I believe mypy will actually *not* support this syntax when run under Python <3.8, so it may not be a good idea to require it.

Reply:
That's probably something to discuss on the issue itself, so let's move to #35.